### DISCUSSION TITLE
blender use CPU instead of GPU

### DISCUSSION THREAD

**Discussion by mikdmst1** [rocm-arch/rocm-arch#939] [Support]
hi all!
i have archlinux and GPU radeon 6800
i was installed rocm-hip-sdk rocm-opencl-sdk via yay, and my GPU shown and checked in Blender HIP tab, but render does only with CPU, and GPU has 0% usage. who can help?

**Comment by tpkessler** [comment]
Did you follow [blender's docs](https://docs.blender.org/manual/en/latest/render/cycles/gpu_rendering.html#gpu-rendering)? You have to select the GPU rendering individually for each scene.
